app/students/[shortcode]/page.tsx

```diff
@@ -154,7 +155,7 @@ const StudentProfilePage = async ({ params }: { params: { shortcode: string } })
             </Flex>
             <Box position="absolute" top="2" right="2">
               <RestrictedAreaStudent showMessage={false} studentId={studentProfile.userId}>
-                <EditStudent prevStudentProfile={studentProfile} />
+                <EditStudent prevStudentProfile={studentProfile} isAdmin={(await auth())!.user.role === "ADMIN"} />
```

Comment:
Use `Role.ADMIN`

Reply:
Resolved in [ab865de](https://github.com/imperial/cpp-connect/pull/88/commits/ab865de7eac50722f9c8c42f0402f3e86edc048b)